src/containers/Root.jsx

```diff
@@ -2,8 +2,8 @@ import ReactGA from 'react-ga4'
 import React, { useEffect, useCallback } from 'react'
 import { useDispatch } from 'react-redux'
 import Header from '@/components/common/Header.jsx'
-import GoogleAnalyticsReporter from '@/components/analytics'
 import Footer from '@/components/common/Footer.jsx'
+import GoogleAnalyticsReporter from '@/components/analytics'
```

Comment:
How this component used?

Reply:
It was included before, but npm lint removed it, so I readded it